src/test/java/api/AuthorizationApi.java

```diff
@@ -0,0 +1,47 @@
+package api;
+
+import com.github.javafaker.Faker;
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.Response;
+import schemas.UserDto;
+
+public class AuthorizationApi extends ApiBase {
+
+    Faker faker = new Faker();
+
+    public Response registerNewUser(UserDto user) {
+        String endPoint = "/Account/v1/User";
+        return postRequest(endPoint, 201, user);
+    }
+
+    public void login(UserDto user) {
+        String endPoint = "/Account/v1/Login";
+        postRequest(endPoint, 200, user);
+    }
+
+    public String generateToken(UserDto user) {
+        String endPoint = "/Account/v1/GenerateToken";
+        Response response = postRequest(endPoint, 200, user);
+
+        JsonPath jsonPath = response.jsonPath();
+        return "Bearer " + jsonPath.getString("token");
+    }
+
+    public boolean isAuthorized(UserDto user) {
+        String endPoint = "/Account/v1/Authorized";
+        return postRequest(endPoint, 200, user).asString().equalsIgnoreCase("true");
```

Comment:
Why you use this here asString().equalsIgnoreCase("true") ?
What does this design do?